version.php

```diff
@@ -47,7 +47,7 @@
     $v_js_includes = md5(microtime());
 } else {
     // Change this number when bumping
-    $v_js_includes = 72;
+    $v_js_includes = 73;
```

Comment:
I'm gonna do a quick bump of this counter in master so it is different from this branch

Reply:
done :)

Reply:
@bradymiller should I hold off from merging? This commit was already merged in master.

Reply:
bring it in :)